homeassistant/components/wled/sensor.py

```diff
@@ -0,0 +1,132 @@
+"""Support for WLED sensors."""
+import logging
+from typing import Callable, List, Union
+
+from homeassistant.components.wled import WLED, WLEDDeviceEntity
```

Comment:
Intra package imports should be relative.

Reply:
🤦‍♂️ old habits, thx